doc/source/dev/development_environment.rst

```diff
@@ -58,9 +58,12 @@ once)::
 
 If you installed Python some other way than conda, first install
 `virtualenv`_ (optionally use `virtualenvwrapper`_), then create your
-virtualenv (named ``numpy-dev`` here) with::
+virtualenv (named ``numpy-dev`` here), activate it, and install all project 
+dependencies with::
 
     $ virtualenv numpy-dev
+    $ source numpy-dev/bin/activate # activate virtual environment
```

Comment:
I'll note that there is no single cross-platform and cross-shell command to activate a virtualenv, this won't work on Windows. But I much prefer to not turn this into a longer venv tutorials, so let's leave it like this for now. I'll do a larger rewrite of all the dev docs the coming week anyway, if things go as planned.